index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+     <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+
+     </head>
+     <body>
+       <header>
+         <div class="head-wrapper">
+           <nav class="nav-bar">
+             
+              <a class="logo"  href="#">My Logo</a>
+            
+             <div class="menu">
+                <div class="toogle-btn"></div>
+                <div class="toogle-btn"></div>
+                <div class="toogle-btn"></div>
+             </div>
+
+           </nav>
+         </div>
+       </header>
+
+       <section>
+        <div class="headline-wrapper">
+           <div class="headline-container">
+             <div class="intro"><h1>I'm Vana<br> Glad to see you!</h1></div>
+             <div class="message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+             <div class="end-message">
+               <h3>LET'S CONNECT</h3>
+            </div>
+          </div>
+
+         <div class="social-icon">
+             <ul class="social-content">
+               <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+               <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+               <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+               <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+               <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+              </ul>
+          </div>
+        </div> 
+       </section> 
+     </body>
```

Comment:
Really well done making improvements from the previous reviewers. I hope you've learned a lot doing this project. You've done really well 🤗👏👏. 
- [ ] [OPTIONAL] just as an optional requirement, You can remove the padding from the `body` tag. 
You just have to add some simple CSS such as this: If you look at the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) you will notice there is no whitespace around the design. just something to notice 👍
```css
body {
  margin: 0;
}
```